strings/en_US/messages.po

```diff
@@ -3441,8 +3441,8 @@ msgstr "Search for Dojos"
 msgid "Detect my location"
 msgstr "Detect my location"
 
-msgid "CoderDojo Account has been created"
-msgstr "CoderDojo Account has been created"
+msgid "Your CoderDojo Account has been created"
```

Comment:
Lower case "a" on account (I'm aware it was already uppercase when you got here)